spec/src/main/asciidoc/core/DeploymentDescriptor.adoc

```diff
@@ -467,11 +467,11 @@ The Container Provider provides tools that
 read and import the information contained in the XML deployment
 descriptor.
 
-All Enterprise Beans 3.2 implementations must support Enterprise Beans
-3.1, Enterprise Beans 3.0, Enterprise Beans 2.1, Enterprise Beans 2.0, and Enterprise Beans 1.1 as well as Enterprise Beans 3.1
-deployment descriptors. The definitions of the Enterprise Beans 3.1 Enterprise Beans 3.0, Enterprise Beans 2.1,
+All Enterprise Beans 4.0 implementations must support Enterprise Beans
+3.1, Enterprise Beans 3.0, Enterprise Beans 2.1, Enterprise Beans 2.0, and Enterprise Beans 1.1 as well as Enterprise Beans 3.2
```

Comment:
Missing Enterprise Beans 3.2?

Reply:
3.2 is mentioned last

Reply:
Based on the way this is worded, I think the last one should be updated to 4.0, and 3.2 added earlier.  Otherwise, it doesn't explicitly say anywhere it must support the 4.0 dd.  The first reference to 4.0 is talking about the implementation, not specifically about the 4.0 dd.

Reply:
I agree with Tracy, comparing this with EJB 3.1 spec, the last version should refer to self.
```suggestion
3.2, Enterprise Beans 3.1, Enterprise Beans 3.0, Enterprise Beans 2.1, Enterprise Beans 2.0, and Enterprise Beans 1.1 as well as Enterprise Beans 4.0
```